config/default.go

```diff
@@ -6,13 +6,13 @@ const DefaultValues = `
 Level = "debug"
 Outputs = ["stdout"]
 
-[Database]
+[SyncDB]
 Database = "postgres"
 User = "test_user"
 Password = "test_password"
 Name = "test_db"
 Host = "localhost"
-Port = "5433"
+Port = "5435"
```

Comment:
```suggestion
Port = "5432"
```
I think it is better to use the default config for db as well